prometheus/Makefile

```diff
@@ -30,11 +45,11 @@ compose-profiles:
 shell:
 	@container=$$(eval "${BIN}/script-wizard choose 'docker exec -it into which container?' 'node-exporter' 'prometheus' 'grafana' 'cadvisor' 'alertmanager' 'alert-test' --default 'grafana'") && make --no-print-directory docker-compose-shell SERVICE=$${container}
 
-.PHONY: local-alertmanager
+.PHONY: local-alertmanager # need summary here
 local-alertmanager:
 	@export ENV_FILE=${ENV_FILE} CONTEXT_INSTANCE=${CONTEXT_INSTANCE} PORT_FORWARD="$$(${BIN}/port_forward ${ENV_FILE} alertmanager 9093)"; if [[ ! "$${PORT_FORWARD}" =~ [0-9]+ ]]; then ${BIN}/fault "Could not get the SSH forwarded port"; else echo "Local port forward established on localhost:$${PORT_FORWARD}"; fi;
 
-.PHONY: test-alert
+.PHONY: test-alert # need summary here
```

Comment:
can you add the summary here or remove the comment altogether

Reply:
fixed in [5c113f7](https://github.com/EnigmaCurry/d.rymcg.tech/pull/234/commits/5c113f7b949509d3e6f63cc83f7fbcdf9a581a14)